doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -566,78 +566,96 @@ The query returns all the entities that match all the matching conditions (AND l
 
 There are two kinds of statements: *unary statements* and *binary statements*.
 
-Binary statements are composed by an attribute name (e.g. "temperature"), an operator and a value
+Binary statements are composed by an attribute path (e.g. "temperature"), an operator and a value
 (whose format depends on the operator), e.g.:
 
 ```
 temperature==50
 temperature<=20
 ```
 
+The attribute path is used to select the *element under evaluation* for the statement. Its general
+syntax consists of an attribute name optionally followed by a *navigation path* using the `.`
+character as delimiter, e.g. "temperature.accuracy.average" refers to the navigation path "accuracy.average"
+within the "temperature" attribute.
+
+How the navigation path is interpreted depends on where the Simple Query Language statement is used:
+
+* If used in a `q` query, then the navigation path refers to a sub-key of the attribute value
+  (assuming that the attribute value is a JSON object). If no navigation path is provided (i.e.
+  the attribute path doesn't have any `.`) then the attribute itself is taken into account.
+* If used in a `mq` query, then the first element of the navigation path is mandatory and it means the
+  name of a metadata of the attribute. Additional elements in the navigation path refer to a sub-key of
+  the metadata value (assuming that the metadata value is a JSON object). If no navigation path is provided
+  (beyond the first element which identifies the metadata name) then the metadata itself
+  is taken into account.
+
 The list of operators (and the format of the values they use) is as follows:
 
 + **Equal**: `==`. This operator accepts the following types of right-hand side:
-    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an attribute
-      with that exact name (temperature) and value (40).
+    + Single element, e.g. `temperature==40`. For an entity to match, it must contain an element
+      under evaluation with that exact name (temperature) and value (40) (or which includes the
+      value, in the case of elements under evaluation which value is an array).
```

Comment:
in the case of elements under evaluation which value is an array ...

Shorter:
in case the element under evaluation is an array